<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>123</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="user">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mode">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generate_image">
       <property name="text">
        <string>GenerateImage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generate_code">
       <property name="text">
        <string>GenerateCode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="history">
       <property name="text">
        <string>History</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_new">
       <property name="text">
        <string>Start New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="previous">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="turnback">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="denseButton">
       <property name="text">
        <string>Dense</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="conv2dButton">
       <property name="text">
        <string>Conv2d</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lstmButton">
       <property name="text">
        <string>LSTM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dropoutButton">
       <property name="text">
        <string>Dropout</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>11</y>
      <width>641</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="codePreview">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
